draft-ietf-quic-transport.md

```diff
@@ -1198,12 +1198,23 @@ The fields of a GOAWAY frame are as follows:
 
 # Packetization and Reliability {#packetization}
 
-The maximum packet size for QUIC is the maximum size of the encrypted payload of
-the resulting UDP datagram.  All QUIC packets SHOULD be sized to fit within the
-path's MTU to avoid IP fragmentation.  The recommended default maximum packet
-size is 1350 bytes for IPv6 and 1370 bytes for IPv4.  To optimize better,
-endpoints MAY use PLPMTUD {{!RFC4821}} for detecting the path's MTU and setting
-the maximum packet size appropriately.
+The maximum packet size for QUIC is the maximum size of the entire UDP payload,
+including the public header, encrypted payload, and any authentication fields.
+All QUIC packets SHOULD be sized to fit within the path's MTU to avoid IP
+fragmentation. To optimize bandwidth efficiency, endpoints MAY use Path MTU
+Discovery ({{!RFC1191}}, {{!RFC1981}}) or Packetization Layer Path MTU
+Discovery ({{!RFC4821}}) for detecting the path's MTU, setting the maximum
+packet size appropriately, and storing the result of previous PMTU
+determinations. In the absence of these mechanisms, the recommended default
+maximum packet size is 1350 bytes for IPv6 and 1370 bytes for IPv4. 
+
+All handshake packets MUST include a PADDING frame if necessary to bring the
```

Comment:
Is this ALL packets?  If the server needs to send multiple handshake packets (that happens often because certificates get bloated), do they really need to pad the 10th packet they send?  Surely the goal has been achieved after sending the first packet at the maximum size.  (The client should only ever send one handshake packet.)